common/djangoapps/util/milestones_helpers.py

```diff
@@ -86,33 +87,46 @@ def set_prerequisite_courses(course_key, prerequisite_course_keys):
             add_prerequisite_course(course_key, prerequisite_course_key)
 
 
-def get_pre_requisite_courses_not_completed(user, enrolled_courses):
+def get_pre_requisite_courses_not_completed(user, enrolled_courses):  # pylint: disable=invalid-name
```

Comment:
Hmmm... while I like your name better, it's used in several places throughout the codebase, so I'm not sure if that's the best idea for this PR.